### PR TITLE
Fixed typos

### PR DESCRIPTION
**Pull request by victorhck**
Fixed typos in spanish translation following the review made by @ekaiz_zarraga

### CODE REVIEW

File: app/assets/javascripts/components/locales/es.jsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  "column.mentions": "Menciones",
+  "column.public": "Público",
+  "tabs_bar.compose": "Redactar",
+  "tabs_bar.home": "Inicio",
+  "tabs_bar.mentions": "Menciones",
+  "tabs_bar.public": "Público",
+  "compose_form.placeholder": "¿En qué estás pensando?",
+  "compose_form.publish": "Publicar",
+  "navigation_bar.settings": "Ajustes",
+  "navigation_bar.public_timeline": "Línea de tiempo pública",
```
Comment (Gargron) [inline app/assets/javascripts/components/locales/es.jsx]: I have just one note. This text will be displayed next to the avatar on the sidebar, could you shorten it somehow so it doesn't break into two lines? The column.public (aka column header) could contain the longer version.

Comment (Gargron) [inline app/assets/javascripts/components/locales/es.jsx]: Yep I mean that line.

**Comment by Gargron** [comment]
Okay, please do a git pull from this repository ([upstream pull](https://help.github.com/articles/configuring-a-remote-for-a-fork/)), I have fixed up some things. Now only missing Doorkeeper and Simple Form localizations.

**Comment by victorhck** [comment]
I'm working now in doorkeeper

**Comment by Gargron** [comment]
Make sure to pull in my changes so I can merge more easily once you're done

**Comment by victorhck** [comment]
I'll try... I hope not to make a big mess!!

**Comment by Gargron** [comment]
I've put what we have so far live, you can switch to Spanish in preferences to check how everything looks.

**Pull request merged**